python/requirements/ml/requirements_rllib.txt

```diff
@@ -11,9 +11,10 @@ kaggle_environments==1.7.11
 # Unity3D testing
 mlagents_envs==0.27.0
 # For tests on PettingZoo's multi-agent envs.
-pettingzoo==1.11.1
+pettingzoo==1.14.0; python_version >= '3.7'
+pettingzoo==1.12.0; python_version < '3.7'
```

Comment:
hey sven, sorry I am still a bit confused, is there any point to install 1.12 for py3.6 at this point? given all the api updates in env/wrappers/pettingzoo_env.py?
does pettingzoo_env.py still work with 1.12? or should we just declare we don't support any versions under 1.14 now?